api/test/e2e/route_online_debug_test.go

```diff
@@ -42,7 +42,7 @@ func TestRoute_Online_Debug_Route_Not_Exist(t *testing.T) {
 		testCaseCheck(tc)
 	}
 	basepath := "http://127.0.0.1:9000/apisix/admin/debug-request-forwarding"
-	request, _ := http.NewRequest("POST", basepath, strings.NewReader(`{"url": "http://172.16.238.30:9080/hello_","method": "GET"}`))
+	request, _ := http.NewRequest("POST", basepath, strings.NewReader(`{"url": "http://172.16.238.30:9080/hello_","method": "GET","protocol": "http"}`))
```

Comment:
can we use `http://172.16.238.30` as a global variable?

Reply:
hard code is not a good way

Reply:
> can we use `http://172.16.238.30` as a global variable?

I tried `127.0.0.1:9080` at first, but test case Run failed
until I tried this internal IP `172.16.238.30`of Docker, the debug forwarding request test cases were passed. 
I didn't know why actually, maybe it has something to do with our Docker test environment

Reply:
@liuxiran maybe define `http://172.16.238.30` as a global constant or variable is the first step. Then we could get to see why `127.0.0.0:9080` is not working. Thanks.

Reply:
> maybe define `http://172.16.238.30` as a global constant or variable is the first step.

yes, that is a better way

Reply:
fixed